src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -171,6 +171,24 @@ Engine::Engine() :
     deviceFullName(getDeviceFullName()),
     specialSetup(new CPUSpecialSetup) {
     set_device_name("CPU");
+    auto proc_type_table = get_proc_type_table();
+    ov::threading::IStreamsExecutor::Config streamsConfig;
+    streamsConfig._name = "CPUStreamsExecutor";
+    streamsConfig._streams = 1;
+    streamsConfig._threads = 1;
+    //Initialize Xbyak::util::Cpu object on Pcore for hybrid cores machine
+    if (proc_type_table[0][MAIN_CORE_PROC] > 0 && proc_type_table[0][EFFICIENT_CORE_PROC] > 0) {
+        streamsConfig._streams_info_table.push_back({1, MAIN_CORE_PROC, 1, 0, 0});
+    }
+    if (!streamsConfig._streams_info_table.empty()) {
+        std::shared_ptr<ov::threading::ITaskExecutor> taskExecutor =
+            std::make_shared<ov::threading::CPUStreamsExecutor>(streamsConfig);
+        std::vector<Task> tasks;
+        tasks.emplace_back([&] {
+            dnnl::impl::cpu::x64::cpu();
+        });
```

Comment:
```suggestion
        std::vector<Task> tasks{dnnl::impl::cpu::x64::cpu};
```
If dnnl::impl::cpu::x64::cpu is free function.

Reply:
updated